pythonforandroid/recipes/hostpython3/__init__.py

```diff
@@ -35,6 +36,10 @@ class Hostpython3Recipe(Recipe):
     '''The default url to download our host python recipe. This url will
     change depending on the python version set in attribute :attr:`version`.'''
 
+    patches = (
+        ('patches/pyconfig_detection.patch', is_version_lt("3.8.3")),
```

Comment:
Now applying to anything lower than 3.8.3. Since this issue was also impacting some recent version of the 3.7 branch I thought it would be easier to go that way than being ultra ranging inside MINOR and PATCH with a complex expression (given MAJOR.MINOR.PATCH)